demo/index.html

```diff
@@ -42,8 +42,9 @@
     </script>
     <script type="module" data-demo-script="true" src="../index.js"></script>
     <script type="module">
-      import { registerComponent } from "../index.js"
-      registerComponent('custom-icon');
+      import { AuroIcon } from '../src/auro-icon.js';
+      import * as RuntimeUtils from '@aurodesignsystem/auro-library/scripts/utils/runtimeUtils.mjs';
+      RuntimeUtils.default.prototype.registerComponent('custom-icon', AuroIcon);
```

Comment:
what's the story behind this `.default.prototype` access pattern? Is that an ESM thing?